resources/qml/VideoNodeTile.qml

```diff
@@ -275,7 +283,7 @@ BaseVideoNodeTile {
         }
 
         drag.onActiveChanged: {
-            view.ensureSelected(tile);
+            view.addToSelection([tile]);
```

Comment:
This changed functionality, I think the new behavior is worse. The new behavior is that if you select one tile and grab/drag another, they both end up selected. I think only the grabbed tile should be selected.